src/newton_ac_powerflow.jl

```diff
@@ -176,7 +177,21 @@ function solve_powerflow!(
     fb = data.power_network_matrix.fb
     tb = data.power_network_matrix.tb
 
+    progress_bar = ProgressMeter.Progress(
```

Comment:
lets not do this in this PR. In the context of the embedded power flow with PSI it will conflict with the PSI progressmetter.